src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx

```diff
@@ -67,10 +45,36 @@ const Repositories = () => {
 }
 
 export const BulkPullReviews = () => {
+  const { translate } = useDictionary()
+  const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
+  const [hasChecked, setHasChecked] = useState<boolean>(false)
```

Comment:
same here, lift the state up

Reply:
no need to add another state if we know the status of `flattenCheckedPulls` in this component.

Reply:
uhm,, you mean.,.,
1. lift state up isDialogOpen 
2. no need hasChecked state

Reply:
I resolved 2. no need hasChecked state.
and is it right that you mean 1. lift state up isDialogOpen?